packages/engine/src/proto.rs

```diff
@@ -226,9 +227,27 @@ pub struct ExperimentRunBase {
 }
 
 impl ExperimentRunBase {
-    /// Returns `true` if the experiment uses Python init or has any Python behavior.
-    pub fn requires_python_runner(&self) -> bool {
-        self.project_base.package_init.initial_state.name == InitialStateName::InitPy
+    /// Returns a `RunnerSpawnConfig` matching the config required by the files present in the
+    /// experiment.
+    pub fn runner_spawn_config(&self) -> RunnerSpawnConfig {
```

Comment:
No strong opinion, but I would have chosen `create_runner_spawn_config`. Feel free to ignore

Reply:
Renamed in https://github.com/hashintel/hash/pull/552/commits/f28e23cc9b846b9740a3f4c009d8bdbba20dd961